x-pack/plugins/security_solution/public/common/components/landing_page/get_started/styles/card_step.styles.ts

```diff
@@ -17,7 +16,7 @@ export const useCardStepStyles = () => {
 
   const customStyles = useMemo(
     () => ({
-      stepPanelStyles: css`
+      stepPanelStyles: `
```

Comment:
Why have the `css` calls been moved to the components? Can't we keep them in the _styles.ts_ files?